evm/src/cpu/kernel/opcodes.rs

```diff
@@ -4,7 +4,7 @@ pub fn get_push_opcode(n: u8) -> u8 {
     0x5f + n
 }
 
-/// The opcode of a standard instruction (not a `PUSH`).
+/// The opcode of a standard instruction (not a `PUSH` or a `MSTORE_32BYTES`).
```

Comment:
I'm not sure this comment is accurate. We do have the different `MSTORE_32BYTES` opcodes, unlike the `PUSH` operations, so I'd leave it unchanged.